nipype/interfaces/fsl/model.py

```diff
@@ -56,7 +56,7 @@ class Level1DesignInputSpec(BaseInterfaceInputSpec):
         desc=("which regressors to make orthogonal e.g., "
               "{1: {0:0,1:0,2:0}, 2: {0:1,1:1,2:0}} to make the second "
               "regressor in a 2-regressor model orthogonal to the first."),
-        default={})
+        value={}, usedefault=True)
```

Comment:
`{}` is already the default.

Reply:
that's true, sometimes I just kept it if author wanted to be explicit, but can remove it